categorize_adjectives/libraries/bert.py

```diff
@@ -0,0 +1,165 @@
+from __future__ import absolute_import
+from __future__ import division
```

Comment:
Add function level comments throughout the file, describing what function does and the parameters.